docs/src/tutorials/introduction.jl

```diff
@@ -140,14 +140,14 @@ end
 # jobs to the GPU and then go do other stuff (such as assigning *more* jobs to the GPU)
 # while the GPU completes its tasks. Wrapping the execution in a `CuArrays.@sync` block
 # will make the CPU block until the queued GPU tasks are done, similar to how `Base.@sync`
-# waits for distributed CPU tasks. Without such a synchronization, you'd be measuring the
+# waits for distributed CPU tasks. Without such synchronization, you'd be measuring the
 # time takes to launch the computation, not the time to perform the computation. But most
 # of the time you don't need to synchronize explicitly: many operations, like copying
 # memory from the GPU to the CPU, implicitly synchronize execution.
 
 # For this particular computer and GPU, you can see the GPU computation was significantly
 # faster than the single-threaded CPU computation, and that the use of CPU threads makes
-# the two competitive with one another. Depending on your hardware you may get different
+# the two compete with one another. Depending on your hardware you may get different
```

Comment:
I think neither compete not competitive make perfect sense here.

> and the use of multiple threads makes the CPU implementation competitive.

Reply:
Yeah, that's better.